### DISCUSSION TITLE
How is to send ETH to another wallet using Synpress?

### DISCUSSION THREAD

**Discussion by philpetrov** [Synthetixio/synpress#965] [General]
Hi! I want to send ETH to another metamask address using Synpress as plugin of Cypress. But i did't ffind locator of button send and method to do it in Synpress framework. Can i do it? I want deposit ETH using metamask to another wallet

**Comment by duckception** [comment]
Hey! We don't have such a method right now. If you really need this, the workaround would be to go to `https://metamask.github.io/test-dapp/`, connect your wallet, scroll down to the bottom, and do a manual tx there which can later be confirmed with `cy.confirmMetamaskTransaction`.

**Reply by philpetrov** [reply]
thanks a lot
